cnames_active.js

```diff
@@ -1406,6 +1406,7 @@ var cnames_active = {
   "han": "han-js.hanlu.im", // noCF
   "hana": "romantic-kare-c84899.netlify.app",
   "hanan": "hanandito.github.io/teddypicker",
+  "handybox": "jacobrdale.github.io/handybox"
```

Comment:
```suggestion
  "handybox": "jacobrdale.github.io/handybox",
```
As the CI error notes, you are missing a comma at the end of your entry